### ISSUE TITLE
Configuration Help

### ISSUE THREAD

**Issue by rudedogg** [emacs-lsp/lsp-sourcekit#5]
I'm trying to configure doom-emacs (develop branch of https://github.com/hlissner/doom-emacs) and lsp-sourcekit, but can't seem to get it working. Here's what I have in my config.el:

```elisp
(after! lsp-sourcekit
  :config
  (setenv "SOURCEKIT_TOOLCHAIN_PATH" "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain")
  (setq lsp-sourcekit-executable "/Users/rudedogg/Desktop/Contrib/sourcekit-lsp/.build/debug/sourcekit-lsp"))
```

But when I try to enable lsp, I get `LSP :: No LSP server for swift-mode.`. I *think* the toolchain/sourcekit-lsp is working with Sublime Text 3 and VS Code, but can't seem to get it going with emacs.

I know this is probably a config issue on my end, and understand if it's out of scope. If so feel free to close.

**Comment by yyoncho** [comment]
Take a look at \*lsp-log\* buffer it should provide some more info.

**Comment by rudedogg** [comment]
@yyoncho Thanks, I actually don't get a *lsp-log* buffer. When I try `M-x lsp` in doom-emacs I get "No lsp server for swift-mode".

To debug I moved my doom-emacs `~/.emacs.d` and started with a plain emacs install. Adding `melpa`, `swift-mode`, `lsp-mode`, and `lsp-sourcekit`. Then I added this to my `~/.emacs.d/init.el`:

```lisp
(require 'lsp-sourcekit)
(setenv "SOURCEKIT_TOOLCHAIN_PATH" "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain")
(setq lsp-sourcekit-executable "/Users/rudedogg/Desktop/Contrib/sourcekit-lsp/.build/debug/sourcekit-lsp")
```

And running `M-x lsp` connects the server and does *stuff*. I didn't bother installing company and figuring out completions, but the issue definitely seems to be with how I am configuring doom-emacs. So I'll work on figuring that out by asking on their Discord.
